sdk/ai/Azure.AI.Projects/samples/Sample16_FineTuning_Supervised.md

```diff
@@ -0,0 +1,694 @@
+# Sample using Supervised Fine-Tuning (SFT) in Azure.AI.Projects
+
+This sample demonstrates how to create and manage supervised fine-tuning jobs using OpenAI Fine-Tuning API through the Azure AI Projects SDK. The file operations are accessed via the ProjectOpenAIClient `GetFineTuningClient` method. Supervised fine-tuning allows you to customize models for specific tasks using labeled training data.
+
+## Supported Models
+Supported OpenAI models: gpt-4o, gpt-4o-mini, gpt-4.1, gpt-4.1-mini
+
+## Prerequisites
+
+- Install the Azure.AI.Projects package.
+- For deployment samples, also install:
+  - `Azure.ResourceManager.CognitiveServices` - For model deployment via Azure Resource Manager
+
+- Set the following environment variables:
+  - `PROJECT_ENDPOINT`: The Azure AI Project endpoint, as found in the overview page of your Azure AI Foundry project.
+  - `MODEL_DEPLOYMENT_NAME`: The name of the model deployment to use for fine-tuning.
```

Comment:
- `TRAINING_FILE_PATH`: the file with training data.
- `VALIDATION_FILE_PATH`: the file with data for model validation.